cylc/flow/cfgspec/globalcfg.py

```diff
@@ -990,12 +990,26 @@
                    the wrapper script.
             ''')
             Conf('global init-script', VDR.V_STRING, desc='''
-                Add a script before the init-script of all jobs on this
-                platform.
+                A per-platform script which is run before the job file is
+                sourced.
+
+                This should be used sparingly to perform any shell
+                configuration that cannot be performed via other means.
+
+                .. versionchanged:: 8.0.0
+
+                   The ``global init-script`` now runs *before* the job file
+                   is sourced which introduces caveats outlined below.
+
+                .. warning::
+
+                   The ``global init-script`` posesses the following caveats
+                   (as comared to the other ``script-*`` configurations:
```

Comment:
```suggestion
                   The ``global init-script`` has the following caveats,
                   as compared to the other task ``script-*`` items:
```
(a "configuration" is the sum total of all config items, IMO)